metadata/metadata_service/api/swagger_doc/type_metadata/badge_delete.yml

```diff
@@ -0,0 +1,53 @@
+Delete badges of a resource
+---
+tags:
+  - 'type_metadata'
+parameters:
+  - name: table_uri
+    in: path
+    type: string
+    schema:
+      type: string
+    required: true
+    example: 'hive://gold.test_schema/test_table1'
+  - name: column_name
+    in: path
+    type: string
+    schema:
+      type: string
+    required: true
+    example: 'beta'
+  - name: type_metadata_path
+    in: path
+    type: string
+    schema:
+      type: string
+    required: true
+    example: 'type/beta'
+  - name: badge_name
+    in: path
+    type: string
+    schema:
+      type: string
+    required: true
+    example: 'beta'
```

Comment:
in a perfect world this could take just the TM key and the badge name. what's stopping that right now?

Reply:
same point about the other endpoints

Reply:
That's how I originally was implementing them, but then I saw in the __init__.py where the paths are defined, it went /table/<table_uri> and then /table/<table_uri>/column/<column_name>, and it seemed like the other apis followed patterns like that (where things fell under /feature, /dashboard, etc) so I was thinking we might want to keep them grouped in that way. But, if that isn't important I'm fine to change it. Just using the key alone would be more straightforward